SADL/AM-CDM-build.sadl

```diff
@@ -1,135 +1,77 @@
-	described by feedstockMaterial (note "Material consumed during build.") with a single value of type Feedstock,
-	described by quantityOfFeedstockMaterialUsed (note "Mass of material used in solidified parts.") with a single value of type Measurement,
-	described by processingMaterial (note "Ancillary material used for build, e.g., environmental gas.") with a single value of type Feedstock,
-	described by buildPlatform with a single value of type BuildPlatform,
-	described by part (note "Parts built from this build. One build can have multiple built parts.") with values of type Part,
-	described by buildPlan (note "Document that describes the build specific design intent and requirements. This may include plate layout, witness coupons, object quantity, position/orientation.") with values of type Document,
-	described by buildParameters (note "Build parameters such as layer thickness, including environment setting") with values of type BuildParameters,
-	described by buildID (note "An identifier for an AM build.") with a single value of type string,
-	described by buildSoftware (note "Software used in build.") with values of type Software,
-	described by buildOrganization (note "Production organization for this build.") with a single value of type Organization,
-	described by buildLayout with a single value of type Document,
-	described by buildJobFile with a single value of type Document,
-	described by amSystem (note "AM system used for this build.") with a single value of type AMSystem,
-	//described by buildSpecificSubsystems (note "equipment that can be interchanged from AMSystem or InSituMonitoringSystem that would be in use for the duration of a build.") with values of type BuildSpecificSubsystem,
-	described by amInSituData (note "Advanced AM in-process monitoring BIG data, is differentiated from processData.") with values of type AMInSituData,
- 	described by buildInterruptions (note "logs of build interruptions") with values of type BuildInterruption,
-	described by buildDuration (note "Recorded duration of build cycle.") with a single value of type Measurement,
-	described by buildLocation (note "Place of an AM process.") with a single value of type Address,
-	described by buildCooldownDuration (note "Time from build completion to cooling to 100 degrees C.") with a single value of type Measurement,
-	described by buildImages(note "Time from build completion to cooling to 100 degrees C.") with values of type Document,
-	described by buildNote (note "Time from build completion to cooling to 100 degrees C.") with a single value of type string.
+AMMaterialInput is a MaterialInputs,
+	described by feedstockMaterial (note "bulk raw material supplied to the additive manufacturing building process.") with a single value of type MaterialStock,
+	described by quantityOfFeedstockMaterialUsed (note "Mass of material used in solidified parts.") with a single value of type Quantity,

```

Comment:
Depending on re-works of Material.sadl, the quantify of feedstockmaterialused may not belong here.

Reply:
Consider for pull request #17